src/pages/CON_ContactMerge.page

```diff
@@ -153,7 +153,7 @@
                                             <td class="slds-row-select">
```

Comment:
Didn't this page also need the `tabStyle="Contact_Merge__tab"` attribute?

Reply:
@force2b thanks, fixed!